orm/models.py

```diff
@@ -454,6 +467,43 @@ async def update(self, **kwargs) -> None:
 
         await self.database.execute(expr)
 
+    async def bulk_update(
```

Comment:
Sorry I should've noticed this earlier, apologies for that.
But maybe a general refactor would be useful here?
There's a lot of nested code here and it's not very readable. What do you think?

Reply:
Yeah I agree with you it needs to be more readable

Reply:
@aminalaee Any updates ?